modules/services/images/src/main/java/au/com/shiftyjelly/pocketcasts/images/icons/StarFull.kt

```diff
@@ -0,0 +1,82 @@
+package au.com.shiftyjelly.pocketcasts.images.icons
+
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.PathFillType.Companion.NonZero
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.graphics.StrokeCap.Companion.Butt
+import androidx.compose.ui.graphics.StrokeCap.Companion.Round
+import androidx.compose.ui.graphics.StrokeJoin
+import androidx.compose.ui.graphics.StrokeJoin.Companion.Miter
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.graphics.vector.ImageVector.Builder
+import androidx.compose.ui.graphics.vector.path
+import androidx.compose.ui.unit.dp
+import au.com.shiftyjelly.pocketcasts.images.PocketCastsIcons
+
+val PocketCastsIcons.StarFull: ImageVector
```

Comment:
Can we remove `PocketCastsIcons` receiver for all star icons to fix this warning?

> Receiver parameter is never used